.circleci/config.yml

```diff
@@ -1,128 +1,127 @@
 version: 2.1
-
 orbs:
   vault: contentful/vault@1
-
-environment: &base-env
-  NPM_CONFIG_PROGRESS: 'false'
+  nx: nrwl/nx@1.6.1
+environment:
+  &a1
+  NPM_CONFIG_PROGRESS: "false"
   NPM_CONFIG_LOGLEVEL: warn
-
-install-deps: &install-deps
+install-deps:
+  &a2
   run:
     name: Install Dependencies
     command: npm ci && npm run bootstrap
-
-build: &build
+build:
+  &a3
   run:
     name: Build
     command: npm run build
-
 executors:
   node-container-14:
     docker:
       - image: cimg/node:14.15.0
-        environment: *base-env
-
+        environment: *a1
   node-container-16:
     docker:
       - image: cimg/node:16.14.0
-        environment: *base-env
-
+        environment: *a1
 commands:
   lint-and-test:
     steps:
       - checkout
       - vault/get-secrets:
-          template-preset: "npm-read"
-      - *install-deps
-      - *build
+          template-preset: npm-read
+      - *a2
+      - *a3
       - run:
           name: Run Linter
           command: npm run lint
       - run:
           name: Run Tests
           command: npm run test
-
   test-built-app:
     steps:
       - checkout
       - vault/get-secrets:
-          template-preset: "npm-read"
-      - *install-deps
-      - *build
+          template-preset: npm-read
+      - *a2
+      - *a3
       - run:
           name: Run Tests
-          command: |
+          command: >
             FORMER_CWD="$(pwd)"
+
             PROJECT=test-$(date +%s)
+
             mkdir "/tmp/${PROJECT}"
+
             cd "/tmp/${PROJECT}"
+
             node "${FORMER_CWD}/packages/contentful--create-contentful-app/lib/index.js" test
+
             cd test
+
             npm t
   build:
     steps:
       - checkout
       - vault/get-secrets:
-          template-preset: "npm-read"
-      - *install-deps
+          template-preset: npm-read
+      - *a2
       - run:
           name: Run build
           command: npm run build
-
   prepare-release:
     steps:
       - checkout
       - vault/get-secrets:
-          template-preset: "npm-read"
+          template-preset: npm-read
       - run: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
-      - run: git config --global user.email "tech-accounts+gh+whydah-gally@contentful.com"
+      - run: git config --global user.email
+          "tech-accounts+gh+whydah-gally@contentful.com"
       - run: git config --global user.name "whydah-gally"
-      - *install-deps
-      - *build
-
+      - *a2
```

Comment:
Why are you changing the names `install-deps` and `build` to `a2` and `a3`?

Reply:
Unfortunately the yml formatter has it's own opinions - fixed!